src/services/emailService.js

```diff
@@ -0,0 +1,56 @@
+const nodemailer = require('nodemailer');
+
+require('dotenv').config();
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
```

Comment:
Consider adding a `secure` option to the transporter configuration. If the SMTP port is 465, set `secure: true`; otherwise, set it to `false`. This ensures that the connection is secure when required.